agent/util-scripts/gold/pbench-list-tools/test-12.00.txt

```diff
@@ -0,0 +1,16 @@
++++ Running test-12.00 pbench-list-tools --help
+
+pbench-list-tools: you specified an invalid option
```

Comment:
With `--help`, this should not say it is an invalid option, should it?

Reply:
Fixed.